ibis/sql/postgres/compiler.py

```diff
@@ -16,7 +16,16 @@
 import ibis.common as com
 import ibis.expr.datatypes as dt
 import ibis.expr.operations as ops
+import ibis.expr.types as ir
 import ibis.sql.alchemy as alch
+# used for literal translate
+from ibis.mapd.operations import (
```

Comment:
This makes the mapd backend a hard dependency of the postgres backend, so we have to find another way to do this. Users shouldn't have to install mapd client libraries to use the postgres backend.

I would suggest making a new subpackage or module (`geo` maybe?) that contains backend independent code related to geo functionality and importing that code into the backends that need it.

Reply:
in PR #1860 I created a subpackage called `common` and I created a module called geospatial there. the module common.py I renamed to _exceptions.py and move that to common subpackage. if you are OK with that I will do it here.